index.html

```diff
@@ -42,6 +42,219 @@ <h2 class="connect">LET'S CONNECT</h2>
+          
+        </div>
+       
+        <div class="descreption" >
+          <p> A daily selection of privately
+              personalized reads; no accounts or 
+              sign-up required.</p>
+          </div>
+
+            <div class="languages">
+              <ul>
+                <li  class="language-tag">html</li>
+                <li  class="language-tag">css</li>
+                <li  class="language-tag">Javascript</li>
+              </ul>
+            </div>
+
+            <button class="see-project">See Project</button>
+          </div>
+       
+      </div>
+      </div>
+    </section>
+      <section class="last-section">
+        <div>
+          <h1 class="intro">About <br />Myself</h1>
+        </div>
+        <div>
+          <p class="about-me">
+            I’m a software developer! I can help you <br />
+            build a product , feature or website Look <br />
+            through some of my work and <br />
+            experience! If you like what you see and <br />
+            have a project you need coded, don’t <br />
+            hestiate to contact me.
+          </p>
+        </div>
+        <div class="contact-me">
+          <h2 class="connect">LET'S CONNECT</h2>
+        </div>
+        <div class="my-social-link">
+          <ul class="buttons">
+          <li><a href=""><img src="/assets/Github.svg"  class="contact-btn" alt="Github" /></li></a>
+          <li><a href=""><img src="/assets/Twitter.svg" class="contact-btn"  alt="Twitter" /></li></a>
+          <li><a href=""><img src="/assets/medium.svg"  class="contact-btn" alt="Medium" /></li></a>
+          <li><a href=""><img src="/assets/Vector.svg"  class="contact-btn" alt="" /></li></a>
+        </ul>
+        </div>
+        <div class="get-my-resume">
+          <button class="see-project resume">Get My resume</button>
+        </div>
+        
+      </section>
+    
+    <div class="grid-container">
+      <div class="my-skills">
+        
+      </div>
+    </div>
+
+    <div class="vector vec1">
+      <ul>
+        <li><h4>Languages</h4></li>
+        <li><img src="/assets/Union.svg" alt=""></li>
+      </ul>
+    </div>
+
+    <div class ="mastring-languages">
+      <ul >
+        <li class="ms-language"><img class="lg-icons" src="/assets/Ellipse 1.png"/>Javascript</li>
+        <li class="ms-language"><img class="lg-icons" src="/assets/HTML.png"/>HTML</li>
+        <li class="ms-language"><img class="lg-icons" src="/assets/Css.png"/>CSS</li>
+      </ul>
+    </div>
+
+    <div class="vector vec2">
+        <ul>
+          <li><h4>Frameworks</h4></li>
+          <li><img src="/assets/Disabled.svg" alt=""></li>
+        </ul>
+    </div>
+
+    <div><hr class="black-line"></div>
+
+    <div class="vector vec3">
+      <ul>
+        <li><h4>Skills</h4></li>
+        <li><img src="/assets/Disabled.svg" alt=""></li>
+      </ul>
+        </div>
+    
+      <div><hr class="black-line"></div>
+        
     </div>
```

Comment:
- [ Required ] As requested by a previous reviewer please make sure that the design of this part of the page is consistent with the `Figma` template provided. Currently, the `margins` are wider than those of the Figma template, please see the screenshot below and rectify this.

## Figma template
![Screenshot from 2023-01-13 19-04-41](https://user-images.githubusercontent.com/16392046/212365569-68c50985-400a-4f87-ae95-3e53bb27757c.png)


## Your design
![Screenshot from 2023-01-13 18-48-42](https://user-images.githubusercontent.com/16392046/212364844-467eb7eb-f2a8-4f88-ba7a-462840e45f26.png)